<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogManual</class>
 <widget class="QDialog" name="DialogManual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="labelSelectedFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>xxyyzz</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="fileContent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEncrypt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDecrypt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
